### PR TITLE
Load AWS plugin on all nodes

### PR DESCRIPTION
**Pull request by thll**
To support a node becoming a leader at runtime, we have to load the
plugin on all nodes. Previously it was only loaded at the master node.

To prevent the AWS inputs to be started on a non-leader node, mark the
inputs as "only one per cluster", so that the input system can take care
of this automatically.

/jenkins-pr-deps Graylog2/graylog2-server#11373,Graylog2/graylog-plugin-enterprise#2797

### CODE REVIEW

**Review by mpfz0r — APPROVED** [review]

**Comment by mpfz0r** [comment]
@thll we cannot merge this into `master` right now. do we want to create a `leader-election` branch here as well?

**Comment by thll** [comment]
> @thll we cannot merge this into `master` right now. do we want to create a `leader-election` branch here as well?

@mpfz0r: either that or we just wait until the leader election branches have been merged. We just shouldn't forget to merge it in the end 😬

**Comment by cameronattard** [comment]
Is there an ETA on when this will be released?

**Comment by mpfz0r** [comment]
@cameronattard this will be part of 4.3 which should be released at the start of May

**Pull request merged**
